bitcoin/src/address/error.rs

```diff
@@ -38,7 +38,7 @@ impl fmt::Display for Error {
 
         match *self {
             WitnessVersion(ref e) => write_err!(f, "witness version construction error"; e),
-            WitnessProgram(ref e) => write_err!(f, "witness program error"; e),
+            SegwitScriptPubkey(ref e) => write_err!(f, "witness program error"; e),
```

Comment:
In b4544017caf0de075813b619607a9119bf78eaa2:

Should update the error message here (and in one other place).

Reply:
Incase you rush like me; you'll liely need to change the docs on the variant to match the error message.